.github/workflows/create-version-change.yml

```diff
@@ -0,0 +1,171 @@
+name: Create Version Change
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: 'The next version to set (e.g., 3.20.0). Leave empty to use default (next minor version)'
+        required: false
+      dry_run:
+        type: boolean
+        description: 'Run in dry-run mode (show what would be done without making changes)'
+        required: false
+        default: false
+      skip_branches:
+        type: boolean
+        description: 'Skip branch creation, only update version files'
+        required: false
+        default: false
```

Comment:
Is there a case in which we would like to only change the version number without creating a new Branch?